espree.js

```diff
@@ -5114,7 +5113,9 @@ function parseClassDeclaration() {
 
     expectKeyword("class");
 
-    id = parseVariableIdentifier();
+    if (lookahead.type === Token.Identifier) {
```

Comment:
I'm going to be out for most of the day taking kids to the zoo,  but may be able to help later!